pkgs/misc/cups/drivers/hll3230cdw/default.nix

```diff
@@ -0,0 +1,41 @@
+{ stdenv, lib, fetchurl, cups, perl, dpkg }:
+
+let
+  version = "1.0.2-0";
+  src = fetchurl {
+    url = "https://download.brother.com/welcome/dlf103925/hll3230cdwpdrv-${version}.i386.deb";
+    sha256 = "9d49abc584bf22bc381510618a34107ead6ab14562b51831fefd6009947aa5a9";
+  };
+
+in
+stdenv.mkDerivation {
+
+  pname = "cups-brother-hll3230cdw";
+  inherit version;
+
+  buildInputs = [ cups perl ];
+  nativeBuildInputs = [ dpkg ];
+
+  inherit src;
+  dontUnpack = true;
+
+  installPhase = ''
+    dpkg-deb -x "$src" "$out"
+    mkdir -p "$out/usr/lib/cups/filter"
+    ln -s "$out/opt/brother/Printers/hll3230cdw/cupswrapper/brother_lpdwrapper_hll3230cdw" \
+      "$out/usr/lib/cups/filter/brother_lpdwrapper_hll3230cdw"
+    mkdir -p "$out/share/cups/model"
+    ln -s "$out/opt/brother/Printers/hll3230cdw/cupswrapper/brother_hll3230cdw_printer_en.ppd" \
+      "$out/share/cups/model/brother_hll3230cdw_printer_en.ppd"
+  '';
+
+  meta = with lib; {
+    description = "Brother HL-L3230CDW printer driver";
+    license = licenses.unfree;
+    sourceProvenance = with sourceTypes; [ binaryNativeCode ];
+    platforms = platforms.linux;
+    homepage = "http://www.brother.com/";
+    downloadPage = "https://support.brother.com/g/b/downloadend.aspx?c=us&lang=en&prod=hll3230cdw_us_eu_as&os=128&dlid=dlf103925_000&flang=4&type3=10283";
```

Comment:
maintainers is missing

Reply:
This has been added in the updated branch.